mobile/src/views/HomeView.jsx

```diff
@@ -34,6 +43,25 @@ const styles = StyleSheet.create({
   vendorList: {
     paddingHorizontal: 20,
   },
+  heading: {
```

Comment:
in the current mockups, the "Discover" title use #454545 gray color instead of black. We can add a darkGray2 color in our Theme file for now and apply it here. I'll confirm with the designers what their intention is